dashboard/app/models/pd/workshop_daily_survey.rb

```diff
@@ -100,5 +89,12 @@ def self.create_placeholder!(user_id:, pd_workshop_id:, day:, form_id:, submissi
         submission_id: submission_id
       )
     end
+
+    # @override
+    def duplicate?
```

Comment:
You also have the :session_id - any reason to use the combo of workshop_id and day instead of just session ID?

Reply:
I took the same attributes we're using for the [uniqueness validation[(https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/workshop_daily_survey.rb#L47). And we did it that way initially because for day 0 session_id is `nil`.